### PR TITLE
Node Restart

### PR DESCRIPTION
**Pull request by mdomasevicius**

### CODE REVIEW

File: src/Components/PowerOffButton/PowerOffButton.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      setIsCountdown(false)
+      // @ts-ignore
+      window.location.reload(true)
+    } catch (e: any) {
+      parseToastError(e)
+    }
+  }
+
+  return (
+    <IconButton style={{ color: '#FFF' }} onClick={() => setShowConfirmation(true)} disabled={showConfirmation}>
```
Comment (LukutasV) [inline src/Components/PowerOffButton/PowerOffButton.tsx:46]: This icon could be updated to better represent that this is a restart feature, currently it seems that its a shutdown button.
image

**Comment by LukutasV** [comment]
When the restart is initiated the text could be updated that the `Restart is in progress...`, from first time use it seemed that I could still somehow cancel it seeing as how it says `Are you sure you want to restart?` text is still there.
image

File: src/Pages/Authenticated/Admin/FeatureToggle/FeatureToggle.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import { selectors } from '../../../../redux/selectors'
+import styles from './FeatureToggle.module.scss'
+import CheckIcon from '@material-ui/icons/Check'
+
+export const FeatureToggle = () => {
+  const { setFeatures } = tequila
+  const config = useSelector(selectors.configSelector)
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+
+  useEffect(() => {}, [])
```
Comment (LukutasV) [inline src/Pages/Authenticated/Admin/FeatureToggle/FeatureToggle.tsx:22]: Doesn't seem to be providing any purpose, probably can be just removed.

**Review by LukutasV — CHANGES_REQUESTED** [review]

**Review by LukutasV — APPROVED** [review]

File: src/Components/PowerOffButton/PowerOffButton.tsx (hunk 1 above)
Comment (mdomasevicius) [reply src/Components/PowerOffButton/PowerOffButton.tsx:46]: This will change with restyle

**Comment by mdomasevicius** [comment]
> When the restart is initiated the text could be updated that the `Restart is in progress...`, from first time use it seemed that I could still somehow cancel it seeing as how it says `Are you sure you want to restart?` text is still there. image

Made it so Cancel button is hidden after clicking "Restart"  
I think any other imporvements should be considered with restyling of nodeUI

**Pull request merged**